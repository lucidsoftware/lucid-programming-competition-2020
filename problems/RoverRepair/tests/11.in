63
11
7
13
6
9
12
14
3
empty
8
10
empty
empty
empty
empty
0
5
empty
empty
empty
empty
empty
empty
empty
empty
empty
empty
empty
empty
empty
empty
1
2
4
empty
empty
empty
empty
empty
empty
empty
empty
empty
empty
empty
empty
empty
empty
empty
empty
empty
empty
empty
empty
empty
empty
empty
empty
empty
empty
empty
empty
empty
