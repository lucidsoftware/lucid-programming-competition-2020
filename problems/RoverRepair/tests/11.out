Swap nodes 0 and 1
